src/Maker/MakeController.php

```diff
@@ -60,6 +62,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                 'route_path' => Str::asRoutePath($controllerClassNameDetails->getRelativeNameWithoutSuffix()),
                 'route_name' => Str::asRouteName($controllerClassNameDetails->getRelativeNameWithoutSuffix()),
                 'twig_installed' => $this->isTwigInstalled(),
+                'use_annotations' => !$input->getOption('no-annotations'),
                 'template_name' => $templateName,
             ]
```

Comment:
If we're going to do this, then you should also add the route to `routes.yaml`. There is a `YamlSourceManipulator` capable of updating YAML files without losing comments, formatting, etc.